packages/react-filerobot-image-editor/src/components/ToolsBar/index.jsx

```
@@ -41,25 +41,46 @@ const ToolsBar = ({ isPhoneScreen }) => {
-              selectTool={selectTool}
-              t={t}
-              isSelected={currentToolId === id}
-            />
-          )
-        );
-      }),
-    [tabTools, currentToolId],
-  );
+    if (currentTabId === TABS_IDS.ANNOTATE) {
+      let orderedToolIds = [];
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)